crates/swc_ecma_transforms_base/src/fixer.rs

```diff
@@ -438,6 +438,12 @@ impl VisitMut for Fixer<'_> {
     }
 
     fn visit_mut_export_default_expr(&mut self, node: &mut ExportDefaultExpr) {
+        if let Expr::Paren(p) = &mut *node.expr {
+            if let Expr::Fn(f) = &mut *p.expr {
```

Comment:
I thnk we need to check for `Expr::Class`, too

Reply:
Thanks a lot !